packages/api/tests/lambdas/test-create-reconciliation-report.js

```diff
@@ -1590,6 +1717,8 @@ test.serial('reconciliationReportForGranuleFiles reports discrepancy of granule
 });
 
 test.serial('reconciliationReportForGranuleFiles does not fail if no distribution endpoint is defined', async (t) => {
+  process.env.DISTRIBUTION_ENDPOINT = 'https://example.com/';
```

Comment:
Shouldn't this be removed? The unit test is testing that it doesn't fail if a distribution endpoint isn't defined